### PR TITLE
zhdanokL

### PR DESCRIPTION
**Pull request by LiudmilaZhdanok**

### CODE REVIEW

**Review by dmitrypleshko — APPROVED** [review]

**Review by olgaz-skiba — APPROVED** [review]

**Review by Alexandr-R-94 — APPROVED** [review]
Lgfm

**Review by Okkss — APPROVED** [review]

File: homework/java/zhdanok_liudmila/lecture2/src/task1/Addiction.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package task1;
```
Comment (kashlyak) [inline homework/java/zhdanok_liudmila/lecture2/src/task1/Addiction.java:1]: domen

File: homework/java/zhdanok_liudmila/lecture2/src/task1/Addiction.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    private double b;
+
+    public Addiction(double a, double b) {
+        this.a = a;
+        this.b = b;
+    }
+
+    public static void calc(double a, double b) {
+        double result = a + b;
+        System.out.println(a + " + " + b + " = " + result);
```
Comment (kashlyak) [inline homework/java/zhdanok_liudmila/lecture2/src/task1/Addiction.java:15]: use log4j

File: homework/java/zhdanok_liudmila/lecture2/src/task1/ArithmeticMain.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+            case "mult":
+                Multiplication.calc(a,b);
+                break;
+            case "sub":
+                Subtraction.calc(a, b);
+                break;
+            case "rem":
+                Reminder.calc(a, b);
+                break;
+        }
```
Comment (kashlyak) [inline homework/java/zhdanok_liudmila/lecture2/src/task1/ArithmeticMain.java:31]: switch() {
    case"":
}

File: homework/java/zhdanok_liudmila/lecture2/src/task1/Addiction.java (hunk 2 above)
Comment (kashlyak) [inline homework/java/zhdanok_liudmila/lecture2/src/task1/Addiction.java:15]: log.info("Expample {},{}, {}", a , b, result);

File: homework/java/zhdanok_liudmila/lecture2/src/task1/Division.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        this.a = a;
+        this.b = b;
+    }
+
+    public static void calc(double a, double b) {
+        if (b != 0.0) {
+            double result = a / b;
+            System.out.println(a + " / " + b + " = " + result);
+        } else
+            System.out.println("Невозможно выполнить деление на 0");
```
Comment (kashlyak) [inline homework/java/zhdanok_liudmila/lecture2/src/task1/Division.java:18]: log4j

Comment (kashlyak) [reply homework/java/zhdanok_liudmila/lecture2/src/task1/Division.java:18]: IlligalArgumentException

File: homework/java/zhdanok_liudmila/lecture2/src/task1/Multiplication.java [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    private double b;
+
+    public Multiplication(double a, double b) {
+        this.a = a;
+        this.b = b;
+    }
+
+    public static void calc(double a, double b) {
+        double result = a * b;
+        System.out.println(a + " * " + b + " = " + result);
```
Comment (kashlyak) [inline homework/java/zhdanok_liudmila/lecture2/src/task1/Multiplication.java:15]: log4j

File: homework/java/zhdanok_liudmila/lecture2/src/task1/ArithmeticMain.java [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+
+public class ArithmeticMain {
+    public static void main(String[] args) throws IOException {
+        System.out.println("Введите два вещественных числа и необходимое арифметическое действие (add/div/mult/sub/rem)");
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
```
Comment (kashlyak) [inline homework/java/zhdanok_liudmila/lecture2/src/task1/ArithmeticMain.java:11]: use Scanner

File: homework/java/zhdanok_liudmila/lecture2/src/task1/Reminder.java [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+    private double b;
+
+    public Reminder(double a, double b) {
+        this.a = a;
+        this.b = b;
+    }
+
+    public static void calc(double a, double b) {
+        double result = a % b;
+        System.out.println(a + " % " + b + " = " + result);
```
Comment (kashlyak) [inline homework/java/zhdanok_liudmila/lecture2/src/task1/Reminder.java:15]: log4j

File: homework/java/zhdanok_liudmila/lecture2/src/task1/Subtraction.java [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+    private double b;
+
+    public Subtraction(double a, double b) {
+        this.a = a;
+        this.b = b;
+    }
+
+    public static void calc(double a, double b) {
+        double result = a - b;
+        System.out.println(a + " - " + b + " = " + result);
```
Comment (kashlyak) [inline homework/java/zhdanok_liudmila/lecture2/src/task1/Subtraction.java:15]: log4j

**Review by kashlyak — COMMENTED** [review]
lgfm

**Review by javaGuruBY — COMMENTED** [review]
Code should be covered by tests!

**Pull request merged**